graphql/version_resolver.go

```diff
@@ -83,6 +83,7 @@ func (r *versionResolver) BuildVariantStats(ctx context.Context, obj *restModel.
 		Variants:                       options.Variants,
 		Statuses:                       options.Statuses,
 		IncludeBuildVariantDisplayName: true,
+		IsMainlineCommit:               !evergreen.IsPatchRequester(utility.FromStringPtr(obj.Requester)),
```

Comment:
Could we say "IsPatch" instead and case on !IsPatch? MainlineCommit is difficult wording since sometimes we use it to mean a commit version (i.e. repotracker version) and sometimes we use it for anything that isn't a patch (i.e. including git tag and etc).